roombasim/pittras/config.py

```diff
@@ -59,3 +59,7 @@
 
 # timeout for transforms
 PITTRAS_TAKEOFF_TRANSFORM_TIMEOUT = 0.2
+
+# distance from the target that the XYZTranslationTask
+# will deem "close enough" in meters
+PITTRAS_XYZ_TRANSLATION_ACCURACY = 0.2
```

Comment:
Picky, but what is the naming convention we are doing for variables? Theoretically this could be used by other people, so I can see an argument for naming things with `PITTRAS` either way. Curious to know the reasoning for doing it thus far.

Reply:
So right now there is a global `roombasim.config` file that contains universal constants. Then we load the `roombasim.pittras.config` file into it which adds all of our specific constants to the global config module.

My rational for using these kind of names was to make clear which constants were our pittras specific constants in the global module. We could also do something like `cfg.PITTRAS.MY_CONSTANT` but that would take two `LOAD_ATTR` commands for something that would be running every update loop. (This probably wouldn't make a huge difference though)

Reply:
If the configuration is namespaced should it be `cfg.PITTRAS` or `cfg.pittras`?

I think it should be `cfg.pittras` if we go that route because `pittras` is already part of the package namespace (`roombasim.pittras`) so it would make more sense as a whole.

Is the configuration file reloaded every loop? If it only contains constants, why would it need to be reloaded?

Reply:
No, the config file is loaded once at the start. But when you write: `cfg.pittras.my_const`, it gets compiled to bytecode like the following:

```
LOAD_GLOBAL    (cfg)
LOAD_ATTR         (pittras)
LOAD_ATTR         (my_const)
```

Whereas, something like `cfg.my_const` is just:

```
LOAD_GLOBAL   (cfg)
LOAD_ATTR        (my_const)
```

On my machine, the former is 50% slower. But both are blazingly fast, so I don't think this would be an issue.